src/main/res/values/strings.xml

```diff
@@ -160,38 +160,38 @@
     <string name="username_missing">Username missing</string>
     <string name="validation_index_error_DESC">Could not validate index.</string>
     <string name="version">Version</string>
     <string name="version_FORMAT">Version %s</string>
     <string name="versions">Versions</string>
-    <string name="waiting_to_start_download">Starting download…</string>
+    <string name="waiting_to_start_download">Downloading…</string>
     <string name="whats_new">What\'s New</string>
     <string name="website">Website</string>
     <string name="prefs_language_title">Language</string>
     <string name="prefs_personalization">Personalization</string>
-    <string name="show_less">Show Less</string>
+    <string name="show_less">Show less</string>
     <string name="latest">Latest</string>
     <string name="explore">Explore</string>
-    <string name="update_all">Update all</string>
-    <string name="installed_applications">Installed applications</string>
-    <string name="sort_filter">Sort &amp; Filter</string>
-    <string name="new_applications">New applications</string>
-    <string name="install_after_sync">Install updates automatically</string>
-    <string name="install_after_sync_summary">Automatically install app updates after syncing repositories</string>
-    <string name="prefs_updated_apps">Number of updated apps to show</string>
+    <string name="update_all">Install all</string>
+    <string name="installed_applications">Installed</string>
+    <string name="sort_filter">Sort and filter</string>
+    <string name="new_applications">New apps</string>
+    <string name="install_after_sync">Auto-install</string>
+    <string name="install_after_sync_summary">Installs new versions of installed new apps when found</string>
+    <string name="prefs_updated_apps">Number of non-new apps to show</string>
     <string name="prefs_new_apps">Number of new apps to show</string>
     <string name="new_repository">New repository</string>
-    <string name="default_tab">Default Tab</string>
+    <string name="default_tab">Default tab</string>
     <string name="pending">Pending</string>
     <string name="installing">Installing</string>
-    <string name="ignore_battery_optimization_title">Ignore Battery Optimization</string>
-    <string name="ignore_battery_optimization_message">Starting Android 12 there\'s restrictions on running foreground services causing the app to crash on downloads. To prevent this, you should turn off the battery optimization</string>
-    <string name="ignore_battery_optimization_not_supported">The device doesn\'t support ignoring battery optimizations!</string>
-    <string name="dialog_refuse">Don\'t need it</string>
-    <string name="dialog_approve">Let\'s do it!</string>
+    <string name="ignore_battery_optimization_title">Battery Optimization</string>
+    <string name="ignore_battery_optimization_message">Battery optimization crashes the app, especially on Android 12+</string>
```

Comment:
```suggestion
    <string name="ignore_battery_optimization_message">Turn of battery optimization for the app to prevent crashing during downloads</string>
```

Reply:
I think the previous one works better (otherwise off misses one f)